src/francesinha.md

```diff
@@ -1,6 +1,6 @@
 # Francesinha
 
-Francesinha is a traditional sandwich from the city of Porto. The francesinha in its special sandwich variation consists of sausage, fresh sausage, ham, cold meat and beef steak, covered with melted cheese.
+Francesinha is a traditional sandwich from the city of Porto. The francesinha in its special sandwich variation consists of linguiça (type of Portuguese sausage), fresh sausage, ham, cold meat and beef steak, covered with melted cheese.
```

Comment:
There are 2 types of sausage?

Reply:
https://en.wikipedia.org/wiki/Lingui%C3%A7a

Reply:
Might make an article for it as well.

Reply:
chad